package.json

```diff
@@ -59,8 +54,8 @@
     "typescript": "3.5.3"
   },
   "lint-staged": {
-    "*.{ts,tsx}$": [
-      "yarn lint.fix",
+    "src/**/*.ts": [
+      "tslint -p tsconfig.build.json",
```

Comment:
Porque?

Reply:
Básicamente en la manera precedente el sistema de linting estaba comprobando el proyecto por entero aunque se cambiaba un file soltero.

:trollface:

Reply:
All right — basically with that command apparently the whole project is always checked. With this new command, only the effective changed files will be checked